h/views/accounts.py

```diff
@@ -341,16 +350,17 @@ def get_when_not_logged_in(self):
             jinja2.Markup(
                 _(
                     "Your account has been activated! "
-                    'You can now <a href="{url}">log in</a> using the password you '
-                    "provided."
-                ).format(url=self.request.route_url("login"))
+                    "You can now log in using the password you provided."
+                ),
             ),
             "success",
         )
 
         self.request.registry.notify(ActivationEvent(self.request, user))
 
-        return httpexceptions.HTTPFound(location=self.request.route_url("index"))
+        return httpexceptions.HTTPFound(
+            location=self.request.route_url("login", _query={"username": user.username})
+        )
```

Comment:
This is after successfully activating an account. Now redirects to login and prefills the username field